### DISCUSSION TITLE
No student documentation accepted -- possible to get manual review?

### DISCUSSION THREAD

**Discussion by NJC12** [community/community#50860] [GitHub Education]
My student coupon is expiring, but I'm still enrolled and have applied to get it renewed. My student ID is undated, but I have submitted a dated transcript that shows I am still enrolled and a proof of enrollment on school letterhead. Each just gets the automated rejection. (I also tried submitted the undated ID, which had the same result.)

Following advice from similar questions, I made my school email my primary account email, wrote a bio, and added a link to a department website that lists me as a student. After reapplying, nothing was different.

Any advice?

**Comment by samus-aran** [comment]
Hi @NJC12 👋 

Here are a few tips to help you with resolving similar rejection reasons.

1. Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

2. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Hope these help. let us know how you get along 👍
